### DISCUSSION TITLE
Not receiving confirmation email

### DISCUSSION THREAD

**Discussion by marreneeea** [mastodon/mastodon#25665] [Q&A]
Hi! I've been trying to register since 9 pm however I haven't received the confirmation email, I've been requesting it over and over again. 

I've already checked spam and trash but I'm not receiving anything. 

Can someone help pls?

**Comment by Mike-E-angelo** [comment]
Attempting to sign up and getting this as well.  I have tried multiple times and it does not appear to be working over the course of the past hour. 😞

**Comment by TheXyloman** [comment]
This has occurred for me as well been several hours and nothing. Also tested with three different email accounts. All the same result.

**Comment by hackmgrm** [comment]
the same question

**Comment by marijn390** [comment]
Same here...

**Comment by Serazer** [comment]
I have the same problem. Following.

**Comment by abelbrencsan** [comment]
I have the same problem…

**Comment by Gankaku** [comment]
Me as well. It's the weekend, and a holiday weekend at that. I think we'll have to maybe give it until Tuesday. Hang in there!

**Comment by Mike-E-angelo** [comment]
FWIW I just got mine at ~10:30am ET so they are starting to kick out it seems.

**Reply by marreneeea** [reply]
I got mine recently, I must have requested it like 10+ times I received a ton of emails about confirmation email

**Comment by Gankaku** [comment]
Got mine too Mike. Thanks for posting that you got yours.
